### DISCUSSION TITLE
RadioButtonGroup value does not show up in RHF DevTools

### DISCUSSION THREAD

**Discussion by dasMoeppi** [react-hook-form/react-hook-form#3635] [Q&A]
Hey guys,

I am not sure whether the behaviour I am facing is intended or not. When I use radio inputs with RHF, the currently selected value does not show up in the RHF DevTools field state. Only when I hit the submit button, the value is updated in there. I have created a basic demo here, it is just two radio buttons and the DevTools. As you can see there, clicking one of the radio buttons does not change the value field.
[Codesandbox](https://codesandbox.io/s/upbeat-christian-zz0yk)

Is this the desired behaviour?

Kind regards
Michael

**Comment by bluebill1049** [comment]
i think you forgot to save the codesandbox.

**Reply by dasMoeppi** [reply]
Sorry, should be saved now!

**Reply by bluebill1049** [reply]
It's correct, it's uncontrolled, unless you watch that input. the re-render will not get trigger at the form level, you can press the update button to retrieve the latest value tho.
